cli/commands/opampagent/config.go

```diff
@@ -0,0 +1,59 @@
+package opampagent
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	corev3 "github.com/sensu/sensu-go/api/core/v3"
+	"github.com/sensu/sensu-go/cli"
+	"github.com/sensu/sensu-go/cli/commands/helpers"
+	"github.com/sensu/sensu-go/cli/elements/list"
+	"github.com/spf13/cobra"
+)
+
+func ListConfigCommand(cli *cli.SensuCli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "config",
+		Short:        "show opamp agent config",
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 0 {
+				cmd.Help()
```

Comment:
linter is complaining

Reply:
Will fix up CI errors once my power returns 👍